src/main/java/Mission.java

```diff
@@ -0,0 +1,99 @@
+public class Mission {
+    enum MissionType {
+        T, D, E;
+    }
+    String content;
```

Comment:
Can consider OOP solution and include visibility conditions, such as public, private to encapsulate the details.